src/components/DateInput/DateInput.js

```diff
@@ -20,8 +20,8 @@ injectGlobal`
 type DateInputProps = {|
   onChange: (value: ?string) => void,
   value: ?string,
-  withTime?: boolean,
-  stretch?: boolean,
+  withTime ?: boolean,
```

Comment:
@ej9x As I know, we don't use space before `?`

Reply:
fixed